modules/external-auth-about.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * authentication/external-auth.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="external-auth-about_{context}"]
+= About direct authentication with an OIDC identity provider
+
+You can enable direct integration with an external OpenID Connect (OIDC) identity provider to issue tokens for authentication. This bypasses the built-in OAuth server and uses the external identity provider directly.
+
+By integrating directly with an external OIDC provider, you can leverage the advanced capabilities of your preferred OIDC provider instead of being limited by the capabilities of the built-in OAuth server. Your organization can manage users and groups from a single interface, while also streamlining authentication across multiple clusters and in hybrid environments. You can also facilitate machine-to-machine workflows and integrate with existing tools and solutions.
```

Comment:
> You can also facilitate machine-to-machine workflows and integrate with existing tools and solutions.

@bergerhoffer , @anjaltelang , sorry I didn't yet have time to verify this machine-to-machine workflow from QE side. Do we need to remove it and mark it as doc hidden TODO comment?